xbmc/platform/android/media/drm/MediaDrmCryptoSession.cpp

```diff
@@ -170,8 +185,17 @@ void CMediaDrmCryptoSession::RestoreKeys(const std::string& keySetId)
   if (m_mediaDrm && keySetId != m_keySetId)
```

Comment:
here better check also m_sessionId to avoid dereference a nullptr
```suggestion
  if (m_mediaDrm && m_sessionId && keySetId != m_keySetId)
```

Reply:
Add the check